libraries/javascript/eddystone-advertising/eddystone-advertising.js

```diff
@@ -0,0 +1,511 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+(() => {
+  'use strict';
+
+  /**
+     Possible Eddystone frame types.
+     @see {@link https://github.com/google/eddystone/blob/master/protocol-specification.md|Protocol Specification}
+     @see {@link https://github.com/google/eddystone/tree/master/eddystone-url|Eddystone-URL}
+     @see {@link https://github.com/google/eddystone/tree/master/eddystone-uid|Eddystone-UID}
+     @see {@link https://github.com/google/eddystone/tree/master/eddystone-tlm|Eddystone-TLM}
+     @readonly
+     @enum {string}
+   */
+  const EddystoneFrameType = {
+    URL: 'url',
+    UID: 'uid',
+    TLM: 'tlm'
+  };
+
+  /**
+     Assigned UUID for Eddystone Beacons
+     @private
+     @constant {string}
+     @default
+   */
+  const EDDYSTONE_UUID = 'FEAA';
+
+  /**
+     Represents the Advertisement being broadcasted.
+     @class
+   */
+  class EddystoneAdvertisement {
+    /**
+       @constructs EddystoneAdvertisement
+       @param {number} id Unique between browser restarts
```

Comment:
Unique _across_ browser restarts or between?  Between implies it can reset after each restart, but if the detail is important I would document it more clearly.